storybook/index.ts

```diff
@@ -1,5 +1,6 @@
-import { getStorybookUI, configure } from "@storybook/react-native"
+import { configure, getStorybookUI } from "@storybook/react-native"
 import "loki/configure-react-native"
+import "./rn-addons"
 
 configure(() => {
   import("../src/stories")
```

Comment:
Probably old-news, but curious why stories have to be manually aggregated 🤔

Reply:
`require.context` doesn't work in a React Native environment so you have to use `import` 😞

Reply:
This sounds vaguely familiar 🙆‍♂